### PR TITLE
Disambiguation of display names

### PR DESCRIPTION
**Pull request by bmarty**
This PR implements adding userId after the display name when there are several members with the same display name in a room.

Prefer using TimelineEvent.getDisambiguatedDisplayName() when appropriate now.

Also "sender" has been renamed to "senderId" for code clarity

### CODE REVIEW

File: matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt [hunk 1]
```diff
@@ -53,6 +54,18 @@ data class TimelineEvent(
         }
     }
 
+    fun getDisambiguatedDisplayName(): String {
+        return if (isUniqueDisplayName) {
+            senderName ?: root.senderId
+        } else {
+            senderName?.let {
+                it + " (" + root.senderId + ")"
```
Comment (ganfra) [inline matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt]: Use string interpolation?

File: matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt [hunk 2]
```diff
@@ -53,6 +54,18 @@ data class TimelineEvent(
...
+            }
+                    ?: root.senderId
```
Comment (ganfra) [inline matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt]: Let the root.senderId be outside of the if as you write it twice.

File: matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt (hunk 1 above)
Comment (bmarty) [reply matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt]: done

File: matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt (hunk 2 above)
Comment (bmarty) [reply matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt]: done

**Pull request merged**